package.json

```diff
@@ -27,6 +27,7 @@
     "enzyme-adapter-react-16": "^1.15.2",
     "express": "^4.17.1",
     "faker": "^4.1.0",
+    "json2csv": "^5.0.1",
     "mongoose": "^5.9.11",
```

Comment:
Are you still using the json2csv module?

Reply:
nope, i'll remove it super quick